packages/patternfly-3/patternfly-react/src/components/LoadingState/LoadingState.stories.js

```diff
@@ -1,35 +1,84 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { withKnobs, boolean, select } from '@storybook/addon-knobs';
+import { withKnobs, boolean, select, number } from '@storybook/addon-knobs';
 import { withInfo } from '@storybook/addon-info';
 import { inlineTemplate } from 'storybook/decorators/storyTemplates';
 import { storybookPackageName, DOCUMENTATION_URL, STORYBOOK_CATEGORY } from 'storybook/constants/siteConstants';
 
 import { LoadingState } from './index';
+import { Button } from '../Button';
 
 import { name } from '../../../package.json';
 
 const stories = storiesOf(`${storybookPackageName(name)}/${STORYBOOK_CATEGORY.COMMUNICATION}/Loading State`, module);
 
 stories.addDecorator(withKnobs);
 
-stories.add(
-  'Loading State',
-  withInfo('Loading State shows centered spinner when loading')(() => {
-    const loading = boolean('Loading', true);
-
-    const size = select('Size', ['lg', 'md', 'sm', 'xs'], 'lg');
-
-    const story = (
-      <LoadingState loading={loading} size={size}>
-        <div>Look at me! I am loaded content!</div>
-      </LoadingState>
-    );
-
-    return inlineTemplate({
-      title: 'Loading State',
-      documentationLink: `${DOCUMENTATION_URL.PATTERNFLY_ORG_COMMUNICATION}loading-state/`,
-      story
-    });
-  })
-);
+stories
+  .add(
+    'Loading State',
+    withInfo('Loading State shows centered spinner when loading')(() => {
+      const loading = boolean('Loading', true);
+
+      const size = select('Size', ['lg', 'md', 'sm', 'xs'], 'lg');
+
+      const story = (
+        <LoadingState loading={loading} size={size}>
+          <div>Look at me! I am loaded content!</div>
+        </LoadingState>
+      );
+
+      return inlineTemplate({
+        title: 'Loading State',
+        documentationLink: `${DOCUMENTATION_URL.PATTERNFLY_ORG_COMMUNICATION}loading-state/`,
+        story
+      });
+    })
+  )
+  .add(
+    'Loading State 2',
+    withInfo('Loading State shows in delay')(() => {
+      class LoadableState extends React.Component {
+        state = {
+          message: 'State 1',
```

Comment:
How about `Pre-Loaded state`? Though I'm not sure showing the message is necessary.